### PR TITLE
Adding primary key to tables

### PR DESCRIPTION
**Pull request by Sarangan0219**
Introducing primary key constraint to the following tables as follows:
-     am_alert_emaillist_details - EMAIL_LIST_ID
-     am_alert_types_values - ALERT_TYPE_ID
-     am_api_scopes- API_ID, SCOPE_ID (Already Resolved)

Fixes (Some tables)  https://github.com/wso2/product-apim/issues/4805

### CODE REVIEW

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql [hunk 1]
```diff
@@ -1496,6 +1496,7 @@ CREATE TABLE IF NOT EXISTS AM_ALERT_TYPES_VALUES (
             ALERT_TYPE_ID INTEGER,
             USER_NAME VARCHAR(256) NOT NULL ,
 	    STAKE_HOLDER VARCHAR(100) NOT NULL ,
+        PRIMARY KEY (ALERT_TYPE_ID),
```
Comment (menakaj) [inline components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql]: These are added in the test db scripts. Please check in the db scripts in **features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/sql**

**Comment by fazlan-nazeem** [comment]
@Krishanx92 this is a table alter. Are we allowing these to this release?

File: components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql (hunk 1 above)
Comment (fazlan-nazeem) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql]: looks like the same ALERT_TYPE_ID is expected to be in multiple rows. so please check whether this is indeed a primary key.

Comment (Sarangan0219) [reply components/apimgt/org.wso2.carbon.apimgt.impl/src/test/resources/dbscripts/h2.sql]: Edited as below : 
AM_ALERT_EMAILLIST_DETAILS - (EMAIL_LIST_ID,EMAIL)
AM_ALERT_TYPES_VALUES - (ALERT_TYPE_ID,USER_NAME,STAKE_HOLDER)

**Pull request merged**